Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
+        private int id = 10;
```

Comment:
Consider renaming `branchName` to follow C# naming conventions, such as `BranchName`, to enhance code readability and maintain consistency.